docs/Use_cases/Air_quality_monitoring_use_case.md

```diff
@@ -0,0 +1,229 @@
+# Air quality monitoring - use case
+
+The Coiote DM and Azure IoT Hub integration lets you create custom use cases with data visualization. Follow the tutorial below to see how to set up an inspiringly complex use case in just a few steps.
+
+## Prerequisites
+
+- An active Azure subscription.
```

Comment:
Just a thought, what do you think about inserting here the links to "How can I obtain Azure or Coiote DM account?" Maybe Azure Account and PowerBI or OpenWeatherMap are simple (the registration is open), but in the CoioteDM case, it is not so simple.